pnpm-lock.yaml

```diff
@@ -0,0 +1,82 @@
+lockfileVersion: 5.3
```

Comment:
I don't think we need a lock file or a package.json at the root of the project? I assume the hasura cli would be installed within the hasura folder right?